app/cartqueue.py

```diff
@@ -22,28 +22,13 @@
 AUTOMATION_CARTS = [
     {
         "type": "StationID",
-        "minute": 5,
+        "minute": 0,
         "max_delta": 6000
     },
     {
-        "type": "StationID",
```

Comment:
instead of removal, this block and the next should be changed to 
`"type": "PSA"`
[Reference this commit](https://github.com/wsbf/ZAutomate/commit/4725289ccffa283c8fbf0de8cb1a7b04427e5bdc)